migrations/9_espumita_ucok.js

```diff
@@ -0,0 +1,57 @@
+const ERC721Patronage_v0 = artifacts.require("ERC721Patronage_v0");
```

Comment:
This comes from master... I will rebase everything and a lot of the noise will disapear...